app/assets/stylesheets/layout/_toolbar.sass

```diff
@@ -36,6 +36,11 @@
     font-weight: normal
     text-decoration: none
 
+@mixin toolbar-element-sizing
+  margin: 0.625rem 0 0
+  height: 40px
+  line-height: 22px
```

Comment:
What is the purpose of this one?
I tend to get nervous when seeing changes to the line-height, since I only saw bad things happen when the line-height was touched.

On the bright side: I can't see any difference when enabling/disabling this rule.